src/getHumanAge.test.js

```diff
@@ -7,4 +7,54 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should be return current array [0, 0]  for input data(0, 0)', () => {
+    expect(getHumanAge(0, 0))
+      .toEqual([0, 0]);
+  });
+
+  test('should be return current array [0, 0]  for input data (14, 14)', () => {
+    expect(getHumanAge(14, 14))
+      .toEqual([0, 0]);
+  });
+
+  test('should be return current array [1, 1]  for input data (15, 15)', () => {
+    expect(getHumanAge(15, 15))
+      .toEqual([1, 1]);
+  });
+
+  test('should be return current array [1, 1]  for input data (16, 16)', () => {
+    expect(getHumanAge(16, 16))
+      .toEqual([1, 1]);
+  });
+
+  test('should be return current array [1, 1]  for input data (23, 23)', () => {
```

Comment:
Replace 'should be return current array [1, 1]' with 'should return correct array [1, 1]' in the test description